Wikipedia/Code/MWKDataStore.m

```diff
@@ -1105,29 +1118,23 @@ - (void)saveArticle:(MWKArticle *)article {
     NSString *path = [self pathForArticle:article];
     NSDictionary *export = [article dataExport];
     NSError *error;
-    BOOL success = [self saveDictionary:export path:path name:@"Article.plist" error:&error];
-    if (!success) {
-        [self postArticleSaveToDiskDidFailNotification:article.url error:error];
-    }
+    [self saveDictionary:export path:path name:@"Article.plist" error:&error];
+    [self postArticleSaveToDiskDidFailNotification:article.url error:error];
```

Comment:
Consider changing the name of `postArticleSaveToDiskDidFailNotification` to reflect the changes made to it. (The name is no longer declarative of what it does.)